Makefile

```diff
@@ -175,6 +175,16 @@ integration: $(MINISHIFT_BINARY)
 	mkdir -p $(INTEGRATION_TEST_DIR)
 	go test -timeout 3600s $(REPOPATH)/test/integration --tags=integration -v -args --test-dir $(INTEGRATION_TEST_DIR) --binary $(MINISHIFT_BINARY) $(GODOG_OPTS)
 
+.PHONY: integration_smoke
```

Comment:
Personally I do not see a reason why to have `@smoke` tag in current form. The only time consuming scenarios which it excludes from basic.feature is `minishift stop`. Other scenarios which this smoke tag excludes are fast - probably being executed in a sec or two. For example: `Scenario: User can enable the anyuid add-on` or `Scenario Outline: User can set, get, view and unset values in configuration file`. And it still uses deployment of application which is the only thing which could save some time here. So now I guess it would save around 20 secs of time.

What is the use case here, when I should use `@smoke`?

Reply:
> Personally I do not see a reason why to have @smoke tag in current form. The only time consuming scenarios which it excludes from basic.feature is minishift stop. Other scenarios which this smoke tag excludes are fast - probably being executed in a sec or two. For example: Scenario: User can enable the anyuid add-on or Scenario Outline: User can set, get, view and unset values in configuration file. And it still uses deployment of application which is the only thing which could save some time here. So now I guess it would save around 20 secs of time.

make sense

> What is the use case here, when I should use @smoke? 

Considering increasing scenario in @basic. i tagged some fixed scenario as @smoke to run the least scenario such as start, ssh, deploy, stop for a quick validation of bits. Mostly the thought behind this is to help the developer to check the bits are not breaking irrespective of new product level feature is added.